L1Trigger/TrackFindingTracklet/src/FitTrack.cc

```diff
@@ -1005,7 +1004,11 @@ void FitTrack::execute(const ChannelAssignment* channelAssignment,
       countFit++;
 
 #ifdef USEHYBRID
```

Comment:
for the record: these lines affect the overall track rate and duplicates:
![L1TK_elec-pu0_ntrkPerSector_all](https://user-images.githubusercontent.com/43122048/162266292-71cf0d04-979b-46e6-a4ba-91bd38e01546.png)
![L1TK_elec-pu0_duplicatefrac_pt](https://user-images.githubusercontent.com/43122048/162266307-d6df0e5c-859e-427c-91d0-34be9889639f.png)